src/pages/academy/grading/subcomponents/GradingSubmissionsTable.tsx

```diff
@@ -198,22 +357,36 @@ const GradingSubmissionTable: React.FC<GradingSubmissionTableProps> = ({ submiss
 
         <Footer>
           <Flex justifyContent="justify-center" spaceX="space-x-3">
+            <Button
+              size="xs"
+              icon={() => <BpIcon icon={IconNames.DOUBLE_CHEVRON_LEFT} />}
+              variant="light"
+              onClick={() => resetPage()}
```

Comment:
I think semantically `setPage(0)` makes more sense here instead of `resetPage` even though they do the same thing.

Reply:
implemented!